yarn.lock

```diff
@@ -3801,13 +3801,6 @@ async-exit-hook@^2.0.1:
   resolved "https://registry.yarnpkg.com/async-exit-hook/-/async-exit-hook-2.0.1.tgz#8bd8b024b0ec9b1c01cccb9af9db29bd717dfaf3"
```

Comment:
These files are part of https://github.com/salesforce/lwc/pull/3484. Once that PR is merged, this PR will be rebased on top of the new master branch.